flask_pydantic_spec/config.py

```diff
@@ -20,22 +21,22 @@ class Config:
     :ivar OPERATION_ID_TYPE: type of operation ID to use in OpenAPI spec
     """
 
-    def __init__(self, **kwargs: Dict[str, Any]) -> None:
+    def __init__(self, **kwargs: dict[str, Any]) -> None:
         self.PATH: str = "apidoc"
         self.FILENAME: str = "openapi.json"
         self.OPENAPI_VERSION: str = "3.1.0"
         self.UI: str = "redoc"
-        self._SUPPORT_UI: Set[str] = {"redoc", "swagger"}
-        self._SUPPORT_MODE: Set[str] = {"normal", "strict", "greedy"}
+        self._SUPPORT_UI: set[str] = {"redoc", "swagger"}
```

Comment:
oh wow, I forgot that this form of type hinting was in Python 3.9 - I thought it came along later!